src/applications/check-in/components/pages/AppointmentDetails/index.jsx

```diff
@@ -40,6 +41,7 @@ const AppointmentDetails = props => {
   const isPhoneAppointment = appointment?.kind === 'phone';
   const { appointmentId } = router.params;
   const isPreCheckIn = app === 'preCheckIn';
+  const { getPreCheckinComplete } = useStorage(false);
```

Comment:
this doesn't work because it is hard set to use the day-of storage keys. It should be `useStorage(isPreCheckIn)`

Reply:
This could be why our little wrapper experiment wasn't working.